### PR TITLE
Export Additional Types

### LINKED ISSUE #220: Export additional types for schema.ts files
* Export `Collection`, `Field` types
* Remove lookup `collection:`

Related to https://github.com/tinacms/tina-graphql-gateway/issues/213

### PR DESCRIPTION
**Pull request by logan-anderson**
Fixes #220

### CODE REVIEW

**Comment by logan-anderson** [comment]
I also renamed `TinaCloudSection` To `TinaCloudCollection` as we are using the name Collection what do you think of this @jeffsee55 ?

I can also revert the tina-cloud-starter `schema.ts`  back to what it used to be as breaking it up into different files was to test if it was working as expected.

File: apps/tina-cloud-starter/.tina/schema.ts [hunk 1]
```diff
@@ -12,112 +12,10 @@ limitations under the License.
...
-                  ],
-                },
-              ],
-            },
-          ],
-        },
-      ],
-    },
-  ],
+  collections: [AuthorCollection, BlogPostCollection, Marketing],
```
Comment (DirtyF) [inline apps/tina-cloud-starter/.tina/schema.ts:20]: Nice!

**Comment by DirtyF** [comment]
Where do we document this?

**Comment by logan-anderson** [comment]
> Where do we document this?

@DirtyF Right now probably no where but it will make more sense to add docs around it once the types are exported (after this PR is merged).

**Review by jeffsee55 — APPROVED** [review]
@logan-anderson can you just bump the package version?

**Comment by DirtyF** [comment]
@logan-anderson now that this has been merged is there any documentation related to these changes?

**Comment by logan-anderson** [comment]
@DirtyF Thanks for the reminder! I will add it to our TODO list.

**Pull request merged**